regular_season_rating_classifier.py

```diff
@@ -0,0 +1,87 @@
+import pandas as pd
+from sklearn.linear_model import LogisticRegression
+from sklearn.model_selection import GridSearchCV
+import numpy as np
+from advanced_stats_starter import get_adv_season_stats
+from submission_simulation import simulate_submit
+
+pd.set_option('display.max_rows', 500)
+pd.set_option('display.max_columns', 500)
+pd.set_option('display.width', 1000)
+
+season_with_adv_stats = pd.read_csv("seasons_results_adv_stats.csv")
+submission = pd.read_csv("input/SampleSubmissionStage1.csv")
+adv_stat_team_season = get_adv_season_stats()
```

Comment:
Loading end of season stats and not regular season. Not sure how it worked :open_mouth:

Reply:
:laughing: the naming is confusing, but i load the season game data with advanced stats calculated (from csv) to train, then I use our other function (`get_adv_season_stats()`) which calculate end of season adv stats for each to predict